app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizardBody.js

```diff
@@ -81,6 +48,7 @@ PlanWizardBody.defaultProps = {
   plansBody: {},
   planWizardGeneralStep: {},
   planWizardVMStep: {},
+  transformationMappings: null,
```

Comment:
We should set this to an empty array to avoid `transformationMappings.<method> is not a function` type errors

Reply:
Good catch, thanks.